azerothcore/azerothcore-wotlk
# fix(DB/gameobject): Quest Secondhand Diving Gear should respawn faster

## Changes Proposed:
-  Increase respawn rate from 15m to 2s

## Issues Addressed:
- If you are not playing solo all alone on the server, this is annoying for others
- Quest is here https://www.wowhead.com/quest=11139/deprecated-secondhand-diving-gear
- No info is available about spawn rates, but it's mentioned there are different spawn locations? (However, not in the DB here)
- I guess I will soon run out of quests like this to fix 



## Tests Performed:
- Applied the SQL and looted the item



## How to Test the Changes:

- `.quest add 11139`
- `.go xyz -2696 -4253 4 1`
- loot and notice it respawn right away for other to loot
- `/dance`

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
